cmip6_omz/datasets.py

```diff
@@ -0,0 +1,162 @@
+# A module to smoothly read in obs data (could probably be handled by an intake catalog...)
+# also this should read stuff from relative paths to links in 'external' folder.
+
+import xarray as xr
+import numpy as np
+import warnings
+
+from cmip6_preprocessing.preprocessing import correct_lon
+
+from omz_tools import convert_o2_ml_l
+
+# TODO: We are planning on integrating this functionality in xgcm at a later point. Check back in time to see if we can refactor this.
+from xarrayutils.utils import dll_dist
+
+
+def reconstruct_areacello(ds):
```

Comment:
I think this should probably live in another module? It should probably also get a test. We can do that in another PR, but should we make an issue so we don't forget?

Reply:
Sounds good to me. I'll post the issue